pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -284,6 +287,10 @@ func newKubeClient(c *pluginConfig) ([]kubeClient, error) {
 	if err != nil {
 		return nil, k8sErr.Wrap(err)
 	}
+	aggregatorClientset, err := newAggregatorClientset(c.KubeConfigFilePath)
```

Comment:
What benefit do we get using the aggregator?

Reply:
APIService is only available in the aggregator clientset, not the regular one.

Reply:
:+1: